src/System.Runtime.Extensions/src/System/IO/Path.Unix.cs

```diff
@@ -183,15 +183,12 @@ public static bool IsPathRooted(string path)
             return path.Length > 0 && path[0] == DirectorySeparatorChar;
         }
 
-        private static unsafe byte[] CreateCryptoRandomByteArray(int byteLength)
+        private static byte[] CreateCryptoRandomByteArray(int byteLength)
         {
             var arr = new byte[byteLength];
-            fixed (byte* buf = arr)
+            if (!Interop.Crypto.GetRandomBytes(arr, arr.Length))
```

Comment:
Are you sure we want to do this?  I'd explicitly decided to use RAND_pseudo_bytes because RAND_bytes will fail if there's not enough entropy available, which is definitely possible, and I don't believe any of these consuming APIs are supposed to fail in such a case.